packages/terra-text/src/Text.jsx

```diff
@@ -63,17 +59,20 @@ const propTypes = {
    * Sets the text size. One of `200`, `300`, `400`, `700`.
    */
   weight: PropTypes.oneOf([200, 300, 400, 700]),
+  /**
+   * Sets an author defined class, to control the colors of the text
```

Comment:
```suggestion
   * Sets an author defined class, to control the colors of the text.
   * 
   * ![IMPORTANT](https://badgen.net/badge//IMPORTANT/blue?icon=github)
   * Adding `--my-app...` CSS variables is required for proper re-themeability when creating your custom color styles _(see included examples)_.
```
\
\
Adding reminder to use CSS variables:

![IMPORTANT](https://badgen.net/badge//IMPORTANT/blue?icon=github) Adding `--my-app...` CSS variables is required for proper re-themeability when creating your custom color styles _(see included examples)_.